src/main/resources/js/sourcegraph-bitbucket.js

```diff
@@ -0,0 +1,24 @@
+AJS.toInit(() => {
+    // Fetch Sourcegraph URL on page load
+    AJS.$.ajax({
+        url: AJS.contextPath() + '/rest/sourcegraph-admin/1.0/',
+        dataType: 'json'
+    }).done(({ url }) => {
+        if (!url) {
+            console.log('No Sourcegraph URL set')
+            return
+        }
+        console.log(url)
+        // If a Sourcegraph URL is set,
+        // inject a <script> tag to fetch the main JS bundle
+        // from the Sourcegraph instance
+        window.SOURCEGRAPH_URL = url
+        window.localStorage.SOURCEGRAPH_URL = url
+        window.SOURCEGRAPH_PHABRICATOR_EXTENSION = true
```

Comment:
It'd be nice to change this to something like `SOURCEGRAPH_NATIVE_EXTENSION` at some point

Reply:
Thanks for pointing this out, I overlooked it. I think there is a way that we can get rid of this global altogether.